docs/packages.md

```diff
@@ -45,17 +45,35 @@ Linux Software Repository for Microsoft Products uses the GPG (GNU Privacy Guard
 
 ## Command examples for using the Linux repository service
 
-- Install a repository configuration: `curl -sSL https://packages.microsoft.com/config/<distribution>/<version>/prod.list | sudo tee /etc/apt/sources.list.d/microsoft-prod.list` (replacing `<distribution>` and `<version>` with the name of the supported Linux distribution and version you wish to use). If you're unsure what distribution and version you are currently running, you can try entering `lsb_release -a` (for any distro that includes the “lsb-release" package) or `cat /etc/os-release` (for any distro that uses systemd).
+The following commands will configure your Linux OS to install packages from packages.microsoft.com. There are instructions for deb-based systems (e.g. Debian, Ubuntu) and rpm-based systems (e.g. Fedora, RHEL).
 
-- Download the Microsoft repository GPG public key: `curl -sSL https://packages.microsoft.com/keys/microsoft.asc | sudo tee /etc/apt/trusted.gpg.d/microsoft.asc`
+If you're unsure what distribution and version you are currently running, you can try entering `lsb_release -a` (for any distro that includes the “lsb-release" package) or `cat /etc/os-release` (for any distro that uses systemd).
 
-- Register the Microsoft repository GPG public key: `sudo dpkg -i packages-microsoft-prod.deb`
+### deb-based Linux distros
```

Comment:
Would it be okay to update this to "### Debian-based Linux distributions"?  *Or is deb-based more commonly used in the Linux community?

Reply:
Yes that works.